src/helpers/colour.test.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env jest */
+
+const util = require('util')
+
+const configPaths = require('../../config/paths.json')
+
+const sass = require('node-sass')
+const sassRender = util.promisify(sass.render)
+
+const sassConfig = {
+  includePaths: [ configPaths.src ],
+  outputStyle: 'compact'
+}
+
+const sassBootstrap = `
+  $govuk-colours-organisations: (
+    'floo-network-authority': (
+      colour: #EC22FF,
+      colour-websafe: #9A00A8
+    ),
+    'broom-regulatory-control': (
+      colour: #A81223
+    )
+  );
+
+  @import "helpers/colour";
+`
+
+describe('@mixin govuk-organisation-colour', () => {
+  it('returns the colour for a given organisation', async () => {
+    const sass = `
+      ${sassBootstrap}
+
+      .foo {
+        background: govuk-organisation-colour('floo-network-authority');
+      }`
+
+    const results = await sassRender({ data: sass, ...sassConfig })
+
+    expect(results.css.toString().trim()).toBe('.foo { background: #EC22FF; }')
+  })
+
+  it('returns the websafe colour for a given organisation if requested', async () => {
+    const sass = `
+      ${sassBootstrap}
+
+      .foo {
+        color: govuk-organisation-colour('floo-network-authority', $websafe: true);
```

Comment:
I think websafe should be default true?

We don't want people using the non websafe unless they really need to.

GOV.UK uses the websafe for example.

Reply:
It depends on what you're using it for surely? 'Non-websafe' would be fine for background and border colours? Conceptually, I'd expect the function to return the 'true' / accurate colour by default.

Reply:
Good defaults are more important than pure concepts, where we can make it harder for people to fail it's always a good idea.